### PR TITLE
Fix check if opam is already installed

### PR DESCRIPTION
**Pull request by panglesd**
We can only merge this if we cut a release very soon after: The README is modified as the install script is not run anymore as a superuser...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ The official guide [Up and Running with OCaml](https://ocaml.org/docs/up-and-run
 This set-up goes through several steps, some of which have to be redone for every new project. The `ocaml-platform` binary, automates and speeds those steps. However, you still need first to install the few dependencies of the OCaml environment, such as a C compiler (such as `gcc`) and other system tools: `bzip2`, `make`, `bubblewrap`, `patch`, `curl` and `unzip`. In most architecture, you can install them using your package manager, for example on Ubuntu or Debian:
 
 ``` sh
-sudo apt install bzip2 make gcc bubblewrap rsync patch curl unzip
+sudo apt install bzip2 make gcc bubblewrap rsync patch curl unzip sudo
```
Comment (Julow) [inline README.md:17]: The line start in `sudo`, I think this is covered.

Comment (panglesd) [reply README.md]: Right.

File: README.md [hunk 2]
```diff
@@ -17,10 +17,10 @@ This set-up goes through several steps, some of which have to be redone for ever
 sudo apt install bzip2 make gcc bubblewrap rsync patch curl unzip
 ```
 
-You can now run the installer script as `root`:
+You can now run the installer script (your user needs to have `sudo` rights):
```
Comment (Julow) [inline README.md:20]: "your user" is weird. Why not say it calls sudo:
```suggestion
You can now run the installer script (which will call `sudo` for the final installation step):
```

**Review by Julow — APPROVED** [review]
Nit pick on the README but otherwise, let's go!

**Comment by panglesd** [comment]
Thanks! Let's wait to be ready for a release before merging (due to the change in the README).

**Review by Julow — APPROVED** [review]
Looks good :)

**Pull request merged**